tools/enso4igv/src/main/resources/org/enso/tools/enso4igv/enso.tmLanguage.json

```diff
@@ -85,6 +85,16 @@
 						"matches": "\\\""
 					}]
 				},
+				{
+					"name": "string.quoted.triple.begin",
+                                        "begin": "^(\\s*)\"\"\"",
+                                        "end": "^(?!\\1\\s+)(?!\\s*$)"
+				},
+				{
+					"name": "string.quoted.triple.middle",
```

Comment:
What if we try?

```suggestion
					"contentName": "string.quoted.triple.middle",
```

![image](https://github.com/enso-org/enso/assets/1436948/0ab10946-9923-4050-bb1e-6daa9632b2cb)

Reply:
`contentName` works well: 0e12937